components/x-teaser/src/AlwaysShowTimestamp.jsx

```diff
@@ -0,0 +1,21 @@
+import { h } from '@financial-times/x-engine'
+import TimeStamp from './TimeStamp'
+import RelativeTime from './RelativeTime'
+import { differenceInCalendarDays } from 'date-fns'
+import { getDateOnly } from '../../x-teaser-timeline/src/lib/date'
+
+/**
+ * Timestamp shown always, the default 4h limit does not apply here
+ * If same calendar day, we show relative time e.g. X hours ago or Updated X min ago
+ * If different calendar day, we show full Date time e.g. June 9, 2021
+ */
+export default (props) => {
+	const localTodayDate = getDateOnly(new Date().toISOString())
+	const dateToCompare = getDateOnly(new Date(props.publishedDate).toISOString())
+
+	if (differenceInCalendarDays(localTodayDate, dateToCompare) >= 1) {
+		return <TimeStamp {...props} />
+	} else {
+		return <RelativeTime {...props} override4hLimit={true} />
+	}
```

Comment:
**thought**: it might be nicer to move this logic into `o-date` as a new [format](https://github.com/Financial-Times/o-date#format-options)? it's fine here though

Reply:
Ooh interesting. This definitely looks like what you are trying to do @rashedul-hassan-app.